packages/core/src/components/graphs/gauge.ts

```diff
@@ -0,0 +1,397 @@
+		this.drawValueNumber();
+		this.drawDelta();
+
+		arcValue.exit().remove();
+
+		// Add event listeners
+		this.addEventListeners();
+	}
+
+	/**
+	 * draws the value number associated with the Gauge component in the center
+	 */
+	drawValueNumber() {
+		const svg = this.getContainerSVG();
+		const options = this.model.getOptions();
+
+		const arcType = Tools.getProperty(options, "gauge", "type");
+		const value = this.getValue();
+		const delta = this.getDelta();
+
+		// Sizing and positions relative to the radius
+		const radius = this.computeRadius();
+
+		const valueFontSize = Tools.getProperty(
+			options,
+			"gauge",
+			"valueFontSize"
+		);
+		// if there is a delta, use the size to center the numbers, otherwise center the valueNumber
+		const deltaFontSize = Tools.getProperty(
+			options,
+			"gauge",
+			"deltaFontSize"
+		);
+
+		// circular gauge without delta should have valueNumber centered
+		let numbersYPosition = 0;
+		if (arcType === GaugeTypes.FULL && !delta) {
+			numbersYPosition = deltaFontSize(radius);
+		}
+
+		// Add the numbers at the center
+		const numbersGroup = DOMUtils.appendOrSelect(
+			svg,
+			"g.gauge-numbers"
+		).attr("transform", `translate(0, ${numbersYPosition})`);
+
+		// Add the big number
+		const valueNumberGroup = DOMUtils.appendOrSelect(
+			numbersGroup,
+			"g.gauge-value-number"
+		).attr("transform", "translate(-10, 0)"); // Optical centering for the presence of the smaller % symbol
+
+		const numberFormatter = Tools.getProperty(
+			options,
+			"gauge",
+			"numberFormatter"
+		);
+		const valueNumber = valueNumberGroup
+			.selectAll("text.gauge-value-number")
+			.data([value]);
+
+		valueNumber
+			.enter()
+			.append("text")
+			.attr("class", "gauge-value-number")
+			.merge(valueNumber)
+			.style("font-size", `${valueFontSize(radius)}px`)
+			.attr("text-anchor", "middle")
+			.text((d) => numberFormatter(d));
+
+		// add the percentage symbol beside the valueNumber
+		const {
+			width: valueNumberWidth
+		} = DOMUtils.getSVGElementSize(
+			DOMUtils.appendOrSelect(svg, "text.gauge-value-number"),
+			{ useBBox: true }
+		);
+
+		DOMUtils.appendOrSelect(valueNumberGroup, "text.gauge-value-symbol")
+			.style("font-size", `${valueFontSize(radius) / 2}px`)
+			.attr("x", valueNumberWidth / 2)
+			.text("%");
+	}
+
+	/**
+	 * adds the delta number for the gauge
+	 */
+	drawDelta() {
+		const self = this;
+		const svg = this.getContainerSVG();
+		const options = this.model.getOptions();
+		const delta = this.getDelta();
+
+		// Sizing and positions relative to the radius
+		const radius = this.computeRadius();
+		const deltaFontSize = delta
+			? Tools.getProperty(options, "gauge", "deltaFontSize")
+			: () => 0;
+		const numberFormatter = Tools.getProperty(
+			options,
+			"gauge",
+			"numberFormatter"
+		);
+		const arrowSize = Tools.getProperty(options, "gauge", "arrowSize");
+		const numberKerning = Tools.getProperty(
+			options,
+			"gauge",
+			"numberKerning"
```

Comment:
I think this is handling the spacing between value and arrow? if so is there a simpler word/term we can rename it to?